push_hono_images.sh

```diff
@@ -59,7 +70,7 @@ then
       IMAGE_NAME="${ECLIPSE_IMAGE_NAME}"
     fi
     echo "pushing image ${IMAGE_NAME}:${TAG} ..."
-    docker push "${IMAGE_NAME}:${TAG}"
+    #docker push "${IMAGE_NAME}:${TAG}"
```

Comment:
this seems to be commented out by mistake ...

Reply:
keeping it as is for now since I don't want to accidentally push anything ;)

I'll comment this back in when we have reached a consensus on how the script should behave.